ethereum/core/src/main/java/org/hyperledger/besu/ethereum/forkid/ForkIdManager.java

```diff
@@ -162,21 +193,34 @@ private long createForkIds() {
     final CRC32 crc = new CRC32();
     crc.update(genesisHash.toArray());
     genesisHashCrc = getCurrentCrcHash(crc);
-    final List<Bytes> forkHashes = new ArrayList<>(List.of(genesisHashCrc));
-    forkBlockNumbers.forEach(
+    final List<Bytes> numberForkHashes = new ArrayList<>(List.of(genesisHashCrc));
```

Comment:
Bit confused by what "number" means here as opposed to just forkHashes?

Reply:
Agree, forkHashes makes more sense. Renamed it.